starter-code/src/functions-and-arrays.js

```diff
@@ -1,100 +1,242 @@
 // Find the maximum
+function maxOfTwoNumbers (first, second) {
+  if(first>second){
+    return first;
+  } else {
+    return second;
+  }
+}
+
+var largest = maxOfTwoNumbers(8, 6);
+console.log(largest);
 
 // Finding Longest Word
+var longestWord = "";
+
+function findLongestWord (words) {
+  
+  for(var i=0; i<words.length; i++){
```

Comment:
le estas diciendo al for que pare cuando i sea menor que la longitud de la array, debería ser que pare cuando sea mayor para que te funcione.